src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,173 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+
+Code generated by Microsoft (R) PSSwagger
+Changes may cause incorrect behavior and will be lost if the code is regenerated.
+#>
+
+<#
+.SYNOPSIS
+    Get or list migration job status.
+
+.DESCRIPTION
+    Returns a list of disk migration jobs.
+
+.PARAMETER ResourceId
+    The resource id.
+
+.PARAMETER Status
+    The parameters of disk migration job status.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER Name
+    The migration job guid name.
+
+.PARAMETER InputObject
+    The input object of type Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob.
+
+.EXAMPLE
+
+    PS C:\> $migrations = Get-AzsDiskMigrationJob -location local
+    PS C:\> $migration = Get-AzsDiskMigrationJob -location local -Name $migrations[0].MigrationId
+#>
+function Get-AzsDiskMigrationJob
+{
+    [OutputType([Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob])]
+    [CmdletBinding(DefaultParameterSetName='DiskMigrationJobs_List')]
```

Comment:
Please only Use 'List', 'Get', and 'ResourceId'

Reply:
OK